addons/sale/data/onboarding_data.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data noupdate="1">
+    <!-- Sale Quotation - ONBOARDING STEPS -->
+    <record id="sale_quotation_onboarding_order_confirmation_step" model="onboarding.onboarding.step">
```

Comment:
Naming remark: guidelines are about "model first, then suffix" (to know what the record contains by looking at its name: it contains an onboarding step, linked to sale_quotation). So it should be "onboarding_onboarding_step_sale_quotation_confirmation", technically.

I let you find in you what is the best naming, achieving the inner light (you can keep it if you prefer, that's not really blocking merge). Just be consistent through various data files :D (won't copy-paste the comment).